src/psyclone/tests/test_files/dynamo0p3/infrastructure/constants_mod.F90

```diff
@@ -54,7 +55,21 @@ module constants_mod
   character(3), parameter :: PRECISION_REAL = '64'
 #endif
 
+  ! Default real kind for r_solver.
+#if (R_SOLVER_PRECISION == 32)
+  integer,      parameter :: r_solver = real32
+  character(3), parameter :: PRECISION_R_SOLVER = '32'
+#elif (R_SOLVER_PRECISION == 128)
+  integer,      parameter :: r_solver = real128
+  character(3), parameter :: PRECISION_R_SOLVER = '128'
+#else
+  integer,      parameter :: r_solver = real64
+  character(3), parameter :: PRECISION_R_SOLVER = '64'
+#endif
+
+
   integer, parameter :: real_type    = 1 !< A parameter used to indicate a real data typa
+  integer, parameter :: r_solver_real_type = 1 !< A parameter used to indicate a r_solver data type
```

Comment:
Does it matter that this has the same value as `real_type`? Or have you just copied this out of the LFRic infrastructure?

Reply:
It's copied. I just checked and can confirm that the values are the same in the LFRic infrastructure.